### DISCUSSION TITLE
测速完成后，显示下载速度全部为0.00

### DISCUSSION THREAD

**Discussion by whynotxw** [XIU2/CloudflareSpeedTest#406] [提问]
### 问题描述

大佬好，这个问题是什么原因呢，我一开始怀疑是开启bbr加速的问题，实际证明不是，还请帮忙看看，谢谢！

### 软件版本

v2.2.4

### 附加截图

![image](https://github.com/XIU2/CloudflareSpeedTest/assets/34786961/52bb3e56-ffba-40b6-953a-fd5aaef1e709)

**Comment by XIU2** [comment]
> Issues 只能反馈软件自身问题，这不属于软件自身问题，因此转到 Discussions。

下载测速结果都是 0.00，说明 CloudflareST 下载测速时全都失败了。

自行排查，这些可能性：
1. 下载测速地址有问题（默认下载测速地址不保证可用性），尝试更换其他下载测速地址
2. 测速的 IP 有问题，导致下载测速连不上
3. 你的网络有问题，比如部分地区运营商可能屏蔽了 Cloudflare CDN 的 IP

**Comment by charSLee013** [comment]
试一下在你的执行命令后面加上 `-url http://cdn.cloudflare.steamstatic.com/steam/apps/5952/movie_max.webm` 改为从steam 上下载文件进行测速看下会不会还是 0MB/s
